static/js/stream_popover.js

```diff
@@ -587,7 +587,7 @@ function with_first_message_id(stream_id, topic_name, success_cb, error_cb) {
     // To ensure this option works reliably at a small latency
     // cost for a rare operation, we just ask the server for the
     // latest message ID in the topic.
-    const data = {
+    const opts = {
```

Comment:
I think for this, the pattern you had in other commits of passing an object directly to `channel.get` was better.

Alternatively, maybe we should actually be renaming what's passed into `success` to be `response` rather than `data`?